heed/src/database.rs

```diff
@@ -52,14 +54,14 @@ use crate::*;
 /// wtxn.commit()?;
 /// # Ok(()) }
 /// ```
-pub struct DatabaseOpenOptions<'e, KC, DC> {
+pub struct DatabaseOpenOptions<'e, KC, DC, C> {
     env: &'e Env,
-    types: marker::PhantomData<(KC, DC)>,
+    types: marker::PhantomData<(KC, DC, C)>,
     name: Option<String>,
     flags: AllDatabaseFlags,
 }
 
-impl<'e> DatabaseOpenOptions<'e, Unspecified, Unspecified> {
+impl<'e> DatabaseOpenOptions<'e, Unspecified, Unspecified, Unspecified> {
```

Comment:
We should probably use the `NoopComparator`, what do you think?

Reply:
Good call! Just updated the code by following your suggestion